source/systems/request.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Request System
+layout: default
+---
+# Request System
+
+As the mayor of a fledgling colony, you are sure to be busy managing your citizens and the multitude of issues that arise. At some point, you’ll have too many citizens to deal with every single one of their demands individually.
+
+Thankfully, there’s the Request System! Citizens will automatically ask for the items they need to perform their jobs, and if those items can be provided by your other citizens, they will do so! 
+
+## Getting Started
+
+The request system is based on two components: the [Warehouse](../../source/buildings/warehouse) — which stores all of your colony’s resources — and the [Couriers](../../source/workers/courier) — who pick up and drop off items between buildings. The Couriers wait in the Warehouse until a request comes in.
+
+When a citizen realizes that they need an item, they’ll first look in their inventory, hut block, and attached hut inventory. If they cannot find the item in these locations, they’ll then make a request automatically. If this item is in the Warehouse, a Courier will then take it from the Warehouse and deliver it to your citizen automatically.
```

Comment:
"attached hut inventory" is kinda confusing, maybe say "racks [link] in their hut"?

Reply:
That's much better. Fixed.